### DISCUSSION TITLE
Probe protection plugin

### DISCUSSION THREAD

**Discussion by andrewmarles** [grblHAL/core#317] [Show and tell]
Posted a first cut of a probe protection plugin in case anyone has any interest or comments.

https://github.com/Expatria-Technologies/grblhal_probe_plugin/

It is not specific to Expatria stuff as it is just using IOPorts for the hard pin.  It does currently rely on an IRQ capable pin for that function though.

**Comment by terjeio** [comment]
Nice! I need to get around to add support for third party plugins in the Web Builder.
